cellxgene_resources/curation_visium.ipynb

```diff
@@ -0,0 +1,227 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import anndata as ad\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import scanpy as sc\n",
+    "from PIL import Image\n",
+    "from scipy import sparse"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Specify the folder that includes these files...\n",
+    "- raw_feature_bc_matrix.h5\n",
+    "- spatial/\n",
+    "  - scalefactors_json.json\n",
+    "  - tissue_hires_image.png\n",
+    "  - tissue_lowres_image.png*\n",
+    "  - tissue_positions_list.csv\n",
+    "\n",
+    "*the lowres image will be deleted but is required for `read_visium()`"
```

Comment:
I think having this url https://www.ebi.ac.uk/biostudies/arrayexpress/studies/E-MTAB-12698 and some further comments that the output files and full res tiff are here would be helpful so someone could run the notebook on these files and see the output.

Reply:
Addressed, mostly in a new notebook header